README.md

```diff
@@ -2,4 +2,119 @@
 
 ## General
 
-The [SAP HANA](https://neo4j.com/) connector is one of Conduit plugins. It provides both, a source and a destination Neo4j connector.
+The [SAP HANA](https://www.sap.com/products/technology-platform/hana/what-is-sap-hana.html) connector is one of Conduit plugins. It provides both, a source and a
+destination Sap Hana connector.
+
+### Prerequisites
+
+- [Go](https://go.dev/) 1.19
+- (optional) [golangci-lint](https://github.com/golangci/golangci-lint) 1.50.1
+
+### How to build it
+
+Run `make build`.
+
+### Testing
+
+Run `make test` to run all the unit and integration tests.
+
+## Source
+
+The DB source connects to the database using the provided connection and starts creating records for each table row
+and each detected change.
+
+### Configuration options
+
+| Name                        | Description                                                                                                                                                                                     | Required                                  | Example                                        |
+|-----------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------|------------------------------------------------|
+| `table`                     | The name of a table in the database that the connector should  read from, by default.                                                                                                           | **true**                                  | users                                          |
```

Comment:
>  The name of a table in the database that the connector should  read from, by default.

The sentence appears to be unfinished or?

Reply:
fixed